process/mock/topicsCheckerMock.go

```diff
@@ -5,6 +5,7 @@ type TopicsCheckerMock struct {
 	CheckValidityCalled func(topics [][]byte) error
 }
 
+// CheckValidity -
```

Comment:
Can we move these mocks (dataCodec + topicsChecker) inside `testscommon/sovereign` ?